contrib/devtools/git-subtree-check.sh

```diff
@@ -1,10 +1,44 @@
 #!/bin/sh
-# Copyright (c) 2015 The Bitcoin Core developers
+# Copyright (c) 2015-2020 The Bitcoin Core developers
 # Distributed under the MIT software license, see the accompanying
 # file COPYING or http://www.opensource.org/licenses/mit-license.php.
 
 export LC_ALL=C
-DIR="$1"
+
+check_remote=0
+while getopts "?hr" opt; do
+  case $opt in
+    '?' | h)
+      echo "Usage: $0 [-r] DIR [COMMIT]"
+      echo "       $0 -?"
+      echo ""
+      echo "Checks that a certain prefix is pure subtree, and optionally whether the"
+      echo "referenced commit is present in any fetched remote."
+      echo ""
+      echo "DIR is the prefix within the repository to check."
+      echo "COMMIT is the commit to check, if it is not provided, HEAD will be used."
+      echo ""
+      echo "-r      Check that subtree commit is present in repository."
+      echo "        To do this check, fetch the subtreed remote first. Example:"
+      echo ""
+      echo "            git fetch https://github.com/bitcoin-core/secp256k1.git"
+      echo "            test/lint/git-subtree-check.sh -r src/secp256k1"
```

Comment:
Since we are at it, let's pivx-fy the help
```suggestion
      echo "            git fetch https://github.com/PIVX-Project/bls-signatures.git"
      echo "            contrib/devtools/git-subtree-check.sh -r src/chiabls"
```

Reply:
sounds good 👍 , squashed it in the last commit.